apps/www/src/app/(root)/user/[id]/page.tsx

```diff
@@ -57,7 +63,7 @@ export default async function page({ params: { id } }: { params: { id: string }
                             {user.bio && <hr className="my-1.5 w-full" />}
                             <span className="text-sm text-black/80">{user.bio}</span>
                         </div>
-                        {currentUser?.id === id && (
+                        {currentUser?.id === result?.id && (
```

Comment:
Result will be defined when it reaches this point. so optional chaining isn't necessary.